Makefile

```diff
@@ -223,11 +219,6 @@ O_FILES       := $(foreach f,$(S_FILES:.s=.o),$(BUILD_DIR)/$f) \
 
 OVL_RELOC_FILES := $(shell $(CPP) $(CPPFLAGS) $(SPEC) | $(SPEC_REPLACE_VARS) | grep -o '[^"]*_reloc.o' )
 
-DISASM_BASEROM := $(BASEROM_DIR)/baserom-decompressed.z64
-DISASM_DATA_FILES := $(wildcard $(DISASM_DATA_DIR)/*.csv) $(wildcard $(DISASM_DATA_DIR)/*.txt)
-DISASM_S_FILES := $(shell test -e $(PYTHON) && $(PYTHON) tools/disasm/list_generated_files.py -o $(EXPECTED_DIR) --config-dir $(DISASM_DATA_DIR))
```

Comment:
should we delete `list_generated_files.py` while we're at it? That also would mean we wouldn't need a separate `file_addresses.py` either, that could optionally be folded into `disasm.py`

Reply:
> should we delete list_generated_files.py

right, that's even the reason I turned the disasm into a shell script, I prefered using `find` over guessing filenames separately

separate scripts for file_addresses.py and disasm.py is fine imo